src/store/wallet/effects/create-multisig/create-multisig.ts

```diff
@@ -45,7 +46,7 @@ export const startCreateKeyMultisig =
 
         resolve(key);
       } catch (err) {
-        console.error(err);
+        dispatch(LogActions.error(`Error create key multisig: ${err}`));
```

Comment:
Variable `err` could be a type object `Error`, not string. Try a simple check before LogActions.

```
    let errorStr;
    if (err instanceof Error) {
      errorStr = err.message;
    } else {
      errorStr = JSON.stringify(err);
    }
     dispatch(LogActions.error(`Error create key multisig: ${errorStr}`));
```